frontend/src/components/editor/chrome/panels/datasources-panel.tsx

```diff
@@ -112,6 +112,10 @@ export const DataSourcesPanel: React.FC = () => {
       case "duckdb":
         code = `_df = mo.sql(f"SELECT * FROM ${table.name} LIMIT 100")`;
         break;
+      case "connection":
+        // TODO: Handle connection source
+        code = `_df = mo.sql("SELECT 1", engine=${table.source_type})`;
```

Comment:
could this be 

```
code = `_df = mo.sql(f"SELECT * FROM ${table.name} LIMIT 100", engine=${table.source_type})`;
```

I guess none will appear in that sidebar for now? so can wait